OpenAPI Specification.json

```diff
@@ -112,15 +112,23 @@
           {
             "name": "distributorId",
             "in": "query",
-            "description": "parameter specifying the Data Provider ID for a Distributor, used to return only the meals for that distributor.If not provided meals for all distributors are returned",
+            "description": "Parameter specifying the Data Provider ID for a Distributor, used to return only the meals for that distributor.If not provided meals for all distributors are returned",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "language",
```

Comment:
The language property is named `lang` in responses. It would be good to be consistent and use `lang` here as well.